app/services/green_lanes/back_link_path.rb

```diff
@@ -0,0 +1,53 @@
+module GreenLanes
+  class BackLinkPath
+    def initialize(params:,
+                   category_one_assessments_without_exemptions:,
+                   category_two_assessments_without_exemptions:)
+      @params = params
+      @category_one_assessments_without_exemptions = category_one_assessments_without_exemptions
+      @category_two_assessments_without_exemptions = category_two_assessments_without_exemptions
+    end
+
+    def call
+      ans = @params[:ans]
+      category = determine_category(ans)
+
+      base_params = {
+        commodity_code: @params[:commodity_code],
+        country_of_origin: @params[:country_of_origin],
+        moving_date: @params[:moving_date],
+      }
+
+      if category == 2 && @category_two_assessments_without_exemptions.empty?
+        # Remvoing the old answers for the back link path
+        ans.delete('2')
+
+        Rails.application.routes.url_helpers
```

Comment:
If you include

` include Rails.application.routes.url_helpers`
you can just use the path helpers 

for example just `new_green_lanes_applicable_exemptions_path`

see `app/services/green_lanes/determine_next_page.rb`